include/miopen/miopen.h

```diff
@@ -470,14 +470,14 @@ typedef enum {
         2, /*!< the operation is getting the minimum value of the reduced elements */
     MIOPEN_REDUCE_TENSOR_MAX =
         3, /*!< the operation is getting the maximum value of the reduced elements */
-    // MIOPEN_REDUCE_TENSOR_AMAX =
-    //    4, /*!< the operation is getting the maximum absolute value of the reduced elements */
-    // MIOPEN_REDUCE_TENSOR_AVG =
-    //    5, /*!< the operation is getting the averaged value of the reduced elements */
-    // MIOPEN_REDUCE_TENSOR_NORM1 =
-    //    6, /*!< the operation is adding the absolute values of the reduced elements */
-    // MIOPEN_REDUCE_TENSOR_NORM2 = 7, /*!< the operation is getting the square root of the sum of
-    //                                   squares of the reduced elements */
+    MIOPEN_REDUCE_TENSOR_AMAX =
```

Comment:
Support in the MIOpenDriver is also necessary. Are you going to implement it in next PR?

Reply:
What do you mean?   I think it is already supported.   Both the driver/reduce_driver.hpp(which include smiopen_ReductionHost.hpp) and test/reduce_test.cpp share the test/cpu_reduce_util.hpp.

Reply:
I've overlooked that.